catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -50,11 +51,68 @@ class NodeViewTableViewDemoCell: UITableViewCell {
   }
 }
 
+class NodeViewTableViewPrimaryDemoCell: UITableViewCell {
+
+  let containedButton = MDCButton()
+
+  override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+    setupContainedButton()
+  }
+
+  required init(coder: NSCoder) {
+    super.init(coder: coder)!
+    setupContainedButton()
+  }
+
+  func setupContainedButton() {
+    containedButton.setTitle("Start Demo", for: .normal)
+    let buttonScheme = AppTheme.globalTheme.buttonScheme
```

Comment:
You'll now need to register for `AppTheme.didChangeGlobalThemeNotificationName` notifications to ensure that this button's theme updates when the app theme changes.

Reply:
Ah, this class already registers for the event. You just have to re-apply the theme in `themeDidChange`.